examples/hosts/app-integration/external-data/src/model/appModel.ts

```diff
@@ -13,7 +14,15 @@ import type { IAppModel, IAppModelEvents, ITaskList } from "../modelInterfaces";
  * responsibilities and functionality.
  */
 export class AppModel extends TypedEventEmitter<IAppModelEvents> implements IAppModel {
-    public constructor(public readonly taskList: ITaskList, container: IContainer) {
+    public constructor(
+        public readonly taskList: ITaskList,
+        container: IContainer,
+        private readonly runtime: IContainerRuntime ) {
         super();
+        this.runtime = runtime;
```

Comment:
This assignment isn't necessary with the member being defined in the constructor param.
```suggestion
```